tests/tests_integration/test_mkm.py

```diff
@@ -19,14 +19,16 @@ def test_response_is_as_expected(self):
             ]
         }
 
-        first_game_received = json_response['game']
+        received_games = json_response['game']
 
-        self.assertEqual(first_game_received, expected_response['game'], 'Game received is not correct')
+        assert received_games == expected_response['game']
 
     def test_sandbox_url(self):
         response = mkm_sandbox.market_place.games()
-        self.assertEqual(response.request.url, 'https://sandbox.mkmapi.eu/ws/v1.1/output.json/games')
```

Comment:
I think it's reasonable in the integration tests to assert the ``status_code == 200``. This enforces the response check and helps you to quickly understand why the test didn't pass. For instance, it's good to know that your integration test in your CI is not passing because the API answered with an ``Internal Server Error (500)`` and not because the ``request.url`` is not defined.

Anyway, check the expected ``status_code`` where it is applicable.